quinn-proto/Cargo.toml

```diff
@@ -13,7 +13,7 @@ workspace = ".."
 [features]
 # NOTE: Please keep this in sync with the feature list in `.github/workflows/codecov.yml`, see
 # comment in that file for more information.
-default = ["rustls-ring", "tracing-log", "bloom"]
+default = ["rustls-ring", "tracing-log", "bloom", "bench"]
```

Comment:
can we remove this from the default features so it doesn't accidentally get activated?